libpod/config.go

```diff
@@ -1,4 +1,4 @@
-package libkpod
+package libpod
```

Comment:
Are we changing this to libpodman eventually?

Reply:
I think we'll be keeping the libpod name

Reply:
Yes podman is the tool, libpod is the library.

Reply:
kk, I was just wondering if we should move to libpodman.  I'm good with the shorter libpod, but didn't know if that would cause confusion or name collision.